libs/pilight/core/json.c

```diff
@@ -1436,3 +1436,26 @@ int json_find_string(JsonNode *object, const char *name, char **out) {
 void json_free(void *a) {
 	free(a);
 }
+
+JsonNode *json_vartype(int vartype) {
+	JsonNode *vartype_array = json_mkarray();
+	if ((vartype & JSON_NULL) != 0) {
```

Comment:
What exactly do you mean? I used the spaces exactly as they're used in the rest of the file - or at least cannot see any difference. Can you please provide a profile for a code formatting tool?